nixos/modules/system/boot/luksroot.nix

```diff
@@ -917,5 +917,17 @@ in
     boot.initrd.postDeviceCommands = commonFunctions + preCommands + concatStrings (mapAttrsToList openCommand postLVM) + postCommands;
 
     environment.systemPackages = [ pkgs.cryptsetup ];
+
+    boot.initrd.emergencyPackages = [ pkgs.cryptsetup ];
```

Comment:
Can we now remove cryptsetup from systemPackages?

Reply:
Why would this mean that? This only governs the packages available on PATH in initrd in case you go to emergency/rescue mode in stage 1